uint8 enabled_type
---
uint8 enabled_type